app/controller/toolbar/CircleSelectionTool.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * This class is the controller for the CircleSelectionController.
+ */
+Ext.define('CpsiMapview.controller.toolbar.CircleSelectionController', {
+    extend: 'Ext.app.ViewController',
+
+    alias: 'controller.cmv_circle_selection_toolbar',
+
+    /**
+     * Handles the click on the `Apply` button by firing
+     * the `circleSelectApply` event of the button that created
+     * the toolbar
+     */
+    handleApply: function () {
+        var me = this;
+        var feat = me.getView().feature;
+        me.getView().fireEvent('circleSelectApply', feat);
+    },
+
+    /**
+     * Handles the click on the `Cancel` button by firing
+     * the `circleSelectCancel` event of the button that created
+     * the toolbar
+     */
+    handleCancel: function () {
+        var me = this;
+        me.getView().fireEvent('circleSelectCancel');
+    },
+
+    /**
+     * Gets the radius of the ol circle feature.
+     */
+    getCurrentRadius: function () {
+        var me = this;
+        var feat = me.getView().feature;
+        var radius;
+        if (feat != null) {
+            var geom = feat.getGeometry();
+            if (geom !== undefined) {
+                // geom.getRadius actually is the diameter
+                // therefore we have to divide by 2
+                radius = geom.getRadius() / 2;
+            }
+        }
+        return radius;
+    },
+
+    /**
+     * Handles changes in the numberField by applying the value
+     * as radius for the ol circle feature
+     * @param {Ext numberField} numberField component that fired the event
```

Comment:
```suggestion
     * @param {Ext.form.field.Number} numberField component that fired the event
```

Reply:
solved in https://github.com/compassinformatics/cpsi-mapview/pull/177/commits/b79cde00e7b4d84cc737d59866c7570cad768ee5